javascript/chrome-debugging

```diff
@@ -0,0 +1,32 @@
+# How to use chrome debugger 
+
+Chrome developer tools are a wonderful way to see what is exactly happening with the code at every iteration. 
+
+### Instructions
+Make a simple html file with Notepad by copying this inside it and saving as .html
+``` 
+<!DOCTYPE html>
+<html>
+<head>
+<title>Hello, HTML!</title>
+<script>
+
+Your code goes here!
+
+</script>
+</head>
+<body>
+<h1>Let's Count to 10 with JavaScript!</h1>
+<p id="theCount"></p>
+</body>
+</html>
+``` 
+![Chrome debugger layout](https://drive.google.com/open?id=0B3Ws6ZqKKpLOT19qU01TTW1SeVk)	
+Open your .html file in chrome and 	from control tools --> more tools choose Developer tools
```

Comment:
Perhaps put this above the image of the developer tools?  Also, it might be worth referring to the keyboard shortcuts (Ctr + Sh + I or Ctr + Cmd + I or F12)